solang-parser/src/doccomment.rs

```diff
@@ -1,27 +1,65 @@
 // SPDX-License-Identifier: Apache-2.0
 
+//! Solidity parsed doc comments.
+//!
+//! See also the Solidity documentation on [natspec].
+//!
+//! [natspec]: https://docs.soliditylang.org/en/latest/natspec-format.html
```

Comment:
```suggestion
//! See also the Solidity documentation on [natspec](https://docs.soliditylang.org/en/latest/natspec-format.html).
```

Markdown links are supported here.

Reply:
That is also a valid markdown link, and makes reading easier IMO.

Reply:
Very nice, I didn't know that